### DISCUSSION TITLE
rendering-math-in-html-mathml-mathml-core-and-asciimath/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#416] [Announcements]
# rendering-math-in-html-mathml-mathml-core-and-asciimath/

In this post I discuss how you can render math in the browser using MathML and MathML Core elements, and show AsciiMath as an easier way of writing equations

https://andrewlock.net/rendering-math-in-html-mathml-mathml-core-and-asciimath/

**Comment by AjayKMehta** [comment]
Another option is [Katex](https://katex.org/).

**Comment by yoelf22** [comment]
Very nice.
How difficult can it be to implement this library within open source editors such as tldraw?
https://github.com/tldraw/tldraw/issues/3887

**Comment by yuttai** [comment]
I'm fine to type my Calculus slides in Microsoft PowerPoint and my proposal to the National Science Foundation in Taiwan in Microsoft Word. Since I don't want to disturb myself by LaTeX programming when I need to think and write...
